it/controllers/agentAppointment/ConfirmAgentAppointControllerISpec.scala

```diff
@@ -17,136 +17,371 @@ import java.util.UUID
+  val allPropertiesTextWelsh = "Maent hefyd wedi’u neilltuo i’ch holl eiddo."
+  val onePropertyTextWelsh = "Maent hefyd wedi’u neilltuo i’ch eiddo."
+  val thisAgentCanTextWelsh = "Gall yr asiant hwn:"
+  val addPropertiesTextWelsh = "ychwanegu eiddo at eich cyfrif"
+  val whatHappensNextTextWelsh = "Beth sy’n digwydd nesaf"
+  val youCanAssignTextWelsh = "Gallwch neilltuo neu ddadneilltuo’r asiant hwn o’ch eiddo trwy reoli eich asiantiaid."
+  val managingAgentsTextWelsh = "reoli eich asiantiaid"
+  val goToHomeTextWelsh = "Ewch i hafan eich cyfrif"
+
+  val headingSelector = "h1"
+  val somePropertiesSelector = "#assigned-to"
+  val allPropertiesSelector = "#assigned-to"
+  val onePropertySelector = "#assigned-to"
+  val thisAgentCanSelector = "#agent-can-text"
+  val addPropertiesSelector = "#agent-can-list > li"
+  val whatHappensNextSelector = "#what-happens-next-title"
+  val youCanAssignSelector = "#what-happens-next-text"
+  val managingAgentsLinkSelector = "#manage-agents-link"
+  val goToHomeSelector = "#go-home-link"
+  val ratingsListSelector = "#rates-for"
+
+  val managingAgentsLinkHref = "/business-rates-property-linking/my-organisation/agents"
+  val goToHomeLinkHref = "/business-rates-dashboard/home"
+
 
   "onPageLoad" should {
-    "return 200 & display correct content (Some properties)" in new TestSetup {
+    "return 200 & display the correct English content when the agent has been assigned to some properties" when {
 
-      await(
-        mockAppointAgentSessionRepository.saveOrUpdate(
-          managingPropertyData.copy(managingPropertyChoice = ChooseFromList.name)))
-      await(
-        mockAppointAgentPropertiesSessionRepository
-          .saveOrUpdate(propertiesSessionData.agentAppointAction.map(_.copy(propertyLinkIds = List("123")))))
+      lazy val document: Document = getDocument(English, "Some")
 
-      lazy val document = getDocument(English)
-      document
-        .getElementById(assignedToId)
-        .text() shouldBe "They have also been assigned to the properties you selected."
+      s"has a title of $titleText" in {
+        document.title() shouldBe titleText
+      }
 
-    }
+      s"has a header of '$headingText'" in {
+        document.select(headingSelector).text shouldBe headingText
+      }
+
+      s"has text on the screen of $somePropertiesText" in {
+        document.select(somePropertiesSelector).text shouldBe somePropertiesText
+      }
+
+      s"has text on the screen of $thisAgentCanText" in {
+        document.select(thisAgentCanSelector).text shouldBe thisAgentCanText
+      }
+
+      s"has a bullet point on the screen of $addPropertiesText" in {
+        document.select(addPropertiesSelector).text shouldBe addPropertiesText
+      }
+
+      s"doesn't not have a bullet point on the screen for the ratings list years" in {
+        document.select(ratingsListSelector).size() shouldBe 0
+      }
```

Comment:
Pretty minor, but throughout this spec you've got a copy and paste error of `doesn't not have` which as a double negative sounds like it **_does_** have but you're testing that it **_doesn't_** have

Reply:
Its not minor, thats horrendous 😅 i'll update this